src/test/java/com/imgix/test/TestAll.java

```diff
@@ -125,11 +125,39 @@ public void testHelperBuildNestedPath() {
               "example/chester" + "#1" + ".png",
               "http://securejackangers.imgix.net/example/chester%231.png?w=500"
             },
+            {
+              "Relative Path With Unencoded Reserved Characters",
+              "example/chester" + "/&$+,:;=?@#.jpg",
+              "http://securejackangers.imgix.net/example/chester/%26%24%2B%2C%3A%3B%3D%3F%40%23.jpg?w=500"
+            },
+            {
+              "Relative Path With Unencoded Reserved Delimiters",
+              "example/chester" + "/ <>[]{}|\\^%.jpg",
+              "http://securejackangers.imgix.net/example/chester/%20%3C%3E%5B%5D%7B%7D%7C%5C%5E%25.jpg?w=500"
+            },
+            {
+              "Relative Path With Unencoded Unicode Characters",
+              "example/chester" + "/ساندویچ.jpg",
+              "http://securejackangers.imgix.net/example/chester/%D8%B3%D8%A7%D9%86%D8%AF%D9%88%DB%8C%DA%86.jpg?w=500"
+            },
             {
               "Absolute Path With Params",
               "/example/chester.png",
               "http://securejackangers.imgix.net/example/chester.png?w=500"
             },
+            // TODO(luis): re-enable encoded path suppport at a later date
+            // {
+            //   "Absolute Path With Encoded Unicode Characters",
+            //
+            // "/example/I%20cann%C3%B8t%20bel%C3%AE%C3%A9v%E2%88%91%20it%20wor%EF%A3%BFs!%20%F0%9F%98%B1",
+            //
+            // "http://securejackangers.imgix.net/example/I%20cann%C3%B8t%20bel%C3%AE%C3%A9v%E2%88%91%20it%20wor%EF%A3%BFs!%20%F0%9F%98%B1?w=500"
+            // },
```

Comment:
Leaving this test commented for clarity's sake. Want to be explicit that encoded unicode chars are not supported in paths unless they are proxy paths

Reply:
I'm a bit confused on this part, was this already supported behavior?

Reply:
Sorry my comment isn't clear. Should say `enable` not `re-enable`. Only wanted to let next contributor know explicitly encoded UNICODE paths are not supported atm.